recipe/meta.yaml

```diff
@@ -38,6 +38,8 @@ requirements:
     - six
     - futures  # [py27]
     - arrow-cpp {{ version }}
+    # empty parquet-cpp metapackage, force old versions to be uninstalled
+    - parquet-cpp 1.5.1
```

Comment:
Should We change this one as well to `>=1.5.1`?

Reply:
Either is fine, but because of the monorepo migration we aren't going to be updating the parquet-cpp conda-forge package anymore

Reply:
However having `parquet-cpp=1.5.1` and a different version number in [.parquetcppversion](https://github.com/apache/arrow/blob/master/cpp/src/parquet/.parquetcppversion) can be confusing.

Reply:
I am indifferent. If you want to make changes here, go for it